skypy/utils/tests/test_completeness_function.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+
+def test_logistic_completeness_function():
+    from skypy.utils.completeness_function import logistic_completeness_function
+
+    # Test that (nb, nm) array is returned if m in (mb, nm) and m95 and m50 in (nb, ) are given
+    m = np.ndarray((5, 50))
+    m[:] = np.linspace(18, 28, 50)
+    m = m.T
+    m50 = np.array([24.7, 25, 23, 23.5, 28])
+    m95 = np.array([24, 23, 22, 22.3, 25])
+    p = logistic_completeness_function(m, m95, m50)
+    assert m.shape == p.shape
+
+    # Test that returned shape is equal to shape of m if it is 1D array and m05 and m50 floats
+    m = np.linspace(18, 28, 50)
+    m50 = 24.7
+    m95 = 24
+    p = logistic_completeness_function(m, m95, m50)
+    assert m.shape == p.shape
+
+    # Test that returned shape is equal to shape of m if it is 2D array and m05 and m50 floats
+    # array
+    m = np.ndarray((5, 50))
+    m[:] = np.linspace(18, 28, 50)
+    m = m.T
+    m50 = 24.7
+    m95 = 24
+    p = logistic_completeness_function(m, m95, m50)
+    assert m.shape == p.shape
+
```

Comment:
Have you checked other corner cases? For example, m50, m95 being arrays and m scalar? If this example does not make physical sense, you should specify it in the docstring.